### PR TITLE
"Getting Started" first draft

### PR DESCRIPTION
**Pull request by mattmanning**
Looking for feedback on language, audience fit and topics covered.

For now I'm using https://github.com/convox-examples/example.git as a placeholder, but would like to use a concrete example instead. The example rails app being worked on would probably be a good fit.

### CODE REVIEW

**Comment by mattmanning** [comment]
[getting started guide](https://trello.com/c/qQ0dRZXH/2424-getting-started-guide)

File: GETTING_STARTED.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+#### Linux
+
+    $ curl https://s3.amazonaws.com/praxis-releases/cli/linux/cx -o /usr/local/bin/cx
+    $ chmod +x /usr/local/bin/cx
+
+## Setting Up Your App Platform
+
+Your applications will run on a private platform called a Rack. While your production Rack will likely run on a cloud infrastructure provider like AWS, you can also install a "local" Rack on your development computer. This makes it easy to achieve dev/prod parity. You can write your application against the Praxis spec without worrying about where your app will actually be running.
+
+### Local Rack
```
Comment (nzoschke) [inline GETTING_STARTED.md]: I'd call these development and production

File: GETTING_STARTED.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+The following steps will guide you through deploying an application. They are the same regardless of where your Rack is running.
+
+### Clone the Example App
+
+    $ git clone https://github.com/convox-examples/example.git
+    $ cd example
+
+### Create the App
+
+    $ cx apps create example
```
Comment (nzoschke) [inline GETTING_STARTED.md]: am I talking to the development rack or the production rack now?

File: GETTING_STARTED.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
+# Getting Started With Convox Praxis
+
+Convox Praxis is a universal infrastructure interface. When you develop and deploy applications using Praxis you completely abstract away concerns about where your application is running. In minutes you can set up a system that has perfect development / production parity and deploy your app to the cloud.
+
+This guide will walk you through installing the Praxis CLI, the Rack deployment platform, and deploying an application.
```
Comment (nzoschke) [inline GETTING_STARTED.md]: This guide will walk you through installing the Praxis CLI and developing and deploying an application.

File: GETTING_STARTED.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
+# Getting Started With Convox Praxis
+
+Convox Praxis is a universal infrastructure interface. When you develop and deploy applications using Praxis you completely abstract away concerns about where your application is running. In minutes you can set up a system that has perfect development / production parity and deploy your app to the cloud.
```
Comment (nzoschke) [inline GETTING_STARTED.md]: When you build applications using Praxis

**Pull request merged**